tests/functional/adapter/test_python_model.py

```diff
@@ -216,6 +216,7 @@ def model(dbt, spark):
 """
 
 
+@pytest.mark.skip(reason="Currently failing as run_started_at is the same across dbt runs")
```

Comment:
To make sure I understand the comment, is run_started_at referring to `model["created_at"]` above? In other words the two `batch_id`s are the same?

Reply:
No the fixture for this test specifies the batch_id as:
```batch_id: '{{ run_started_at.strftime("%Y-%m-%d-%H-%M-%S") }}-python-array'```
`run_started_at` appears to be the same for both runs (I'm guessing because of how we setup our integ test runs). We need to come up with a way to specify a different batch_id for different runs